app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -178,6 +184,20 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         false
     )
 
+    fun getWallpaper(): Wallpapers {
+        return Wallpapers.valueOf(currentWallpaper)
+    }
```

Comment:
Looking at the return type here, I think the singular `Wallpaper` would work better for the name of the `enum`.

Reply:
👍🏽